src/sentry/receivers/onboarding.py

```diff
@@ -63,7 +63,7 @@
 def record_new_project(project, user=None, user_id=None, **kwargs):
     if user_id is not None:
         default_user_id = user_id
-    elif user.is_authenticated:
+    elif user and user.is_authenticated:
```

Comment:
Small bug fix, if `user` and `user_id` are both None then this will error. We need to check if `user` exists first